specification/schemaregistry/SchemaRegistry/examples/2023-07-01/ListSchemaGroups.json

```diff
@@ -1,12 +1,14 @@
 {
+  "operationId": "ListSchemaGroups",
+  "title": "List Schema Groups",
   "parameters": {
     "endpoint": "mynamespace.servicebus.windows.net",
     "api-version": "2023-07-01"
   },
   "responses": {
     "200": {
       "body": {
-        "schemaGroups": [
```

Comment:
Same here, if the service actually returns `schemaGroups` then there's probably a rename somewhere that is being applied incorrectly

Reply:
I believe the service doesn't actually return schemaGroups here. We just don't use the list operations in the public api, so I think they've just never been tested before. When I called the operation locally and checked the response from the service, it didn't contain schemaGroups.